src/Debugging/Drivers/ErrorLogDebugger.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Saloon\Debugging\Drivers;
+
+use Saloon\Debugging\DebugData;
+
+class ErrorLogDebugger implements DebuggingDriver
+{
+    public function name(): string
+    {
+        return 'error_log';
+    }
+
+    /**
+     * @param \Saloon\Debugging\DebugData $data
+     *
+     * @return $this
+     */
+    public function send(DebugData $data): static
+    {
+        error_log(print_r($this->formatData($data), true), LOG_DEBUG);
+
+        return $this;
+    }
+
+    /**
+     * @return array<string, mixed>
+     */
+    protected function formatData(DebugData $data): array
+    {
+        // TODO: Format the $data, and send it appropriately to the error_log().
+
+        $formattedData = [
+            'method' => $data->method(),
+            'uri' => $data->url(),
+            'request_headers' => $data->pendingRequest()->headers(),
+            'request_query' => $data->pendingRequest()->query()->all(),
+            'request_payload' => $data->pendingRequest()->body()?->all(),
```

Comment:
We've gotta be careful as some bodies like MultipartBody will return an array of MultipartValue objects which may or may not play nicely with debuggers, this is one of the reasons why I wanted to output the sender's response body because that would be the actual string version of what it should be which IMO is better for debugging.